pages/linux/qmrestore.md

```diff
@@ -1,8 +1,20 @@
 # qmrestore
 
-> Restore QemuServer vzdump Backups.
+> Restore QemuServer vzdump backups.
 > More information: <https://pve.proxmox.com/pve-docs/qmrestore.1.html>.
 
-- Restore KVM-based virtual machine to local storage:
+- Restore virtual machine from given backup file on the original storage:
 
-`qmrestore {{/var/lib/vz/dump/backup_file.vma.lzo}} {{vm_id}} --storage {{local}}`
+`qmrestore {{path/to/vzdump-qemu-100-2022_10_07-09_10_10.vma.lzo}} {{100}}`
```

Comment:
```suggestion
`qmrestore {{path/to/vzdump-qemu-100.vma.lzo}} {{100}}`
```

It is not necessary to add the date string in the file path. We all know it is the file path here.

Reply:
Why not just `path/to/backup`?
```suggestion
`qmrestore {{path/to/backup.vma.lzo}} {{100}}`
```

Reply:
I decided to go with @einverne because I think it's more elegant to keep the VM ID pattern alive in the filepath for better understanding.